### PR TITLE
Update UI when user edits their account

### PR DESCRIPTION
**Pull request by notsrg**

### CODE REVIEW

**Comment by notsrg** [comment]
@hhaidar

File: media/js/views/client.js [hunk 1]
```diff
@@ -56,7 +56,8 @@
                 model: this.client.user
             });
             this.accountModal = new window.LCB.AccountModalView({
-                el: this.$el.find('#lcb-account')
+                el: this.$el.find('#lcb-account'),
+                model: this.client.user
 			});
```
Comment (hhaidar) [inline media/js/views/client.js]: this indentation is not gangster

**Comment by hhaidar** [comment]
One thing, then ship homeboy

File: media/js/views/client.js (hunk 1 above)
Comment (notsrg) [inline media/js/views/client.js]: @hhaidar it's post-modern indentation... I thought you would appreesh

**Comment by notsrg** [comment]
@hhaidar g2g

**Pull request merged**